domains/Accounts/Controllers/AccountsStoreController.php

```diff
@@ -11,13 +11,6 @@
 
 class AccountsStoreController extends Controller
 {
-    private User $user;
-
-    public function __construct(User $users)
-    {
-        $this->users = $users;
-    }
-
```

Comment:
Why is this being removed?

Reply:
To be aligned with the rest of the code, but then decided to keep this injection instead of using `$user = new User();`
in the `__invoke() `itself.
I'll update.